### ISSUE TITLE
Travis-Ci fails using GHC 8.2.1

### ISSUE THREAD

**Issue by jonaprieto** [jonaprieto/athena#54]
Using this travis fails (something related with cabal-install)

```
  include:

    - env: TEST=MAIN GHC_VER=8.2.1 BUILD=CABAL CABAL_VER=1.24
      addons:
        apt:
          packages:
            - alex-3.1.7
            - cabal-install-1.24
            - ghc-8.2.1
            - happy-1.19.5
            - texlive-binaries
          sources:
            - hvr-ghc
```

**Comment by jonaprieto** [comment]
@asr , Do you know how can we fix this in Travis?

**Comment by asr** [comment]
I use `cabal-install-2.0` instead of `cabal-install-1.24` with GHC 8.2.1. Does it fix the issue?

**Comment by jonaprieto** [comment]
No. Loot at for example this https://travis-ci.org/jonaprieto/athena/builds/310473303

**Comment by asr** [comment]
Why are you reinstalling `cabal-install` in `.travis.yml`?

**Comment by jonaprieto** [comment]
I don't remember well. I'll try to remove it and see what happen.

**Comment by jonaprieto** [comment]
Thanks @asr that solved the problem.
Closed by 6c95f15
